--- origsrc/celt-0.5.0/libcelt/Makefile.am	2008-10-09 06:25:57.000000000 -0500
+++ src/celt-0.5.0/libcelt/Makefile.am	2008-10-28 15:40:25.057161000 -0500
@@ -21,7 +21,7 @@
 
 #noinst_HEADERS =
 
-libcelt_la_LDFLAGS = -version-info @CELT_LT_CURRENT@:@CELT_LT_REVISION@:@CELT_LT_AGE@
+libcelt_la_LDFLAGS = -version-info @CELT_LT_CURRENT@:@CELT_LT_REVISION@:@CELT_LT_AGE@ -no-undefined
 
 noinst_HEADERS = _kiss_fft_guts.h arch.h bands.h fixed_c5x.h fixed_c6x.h \
 	cwrs.h ecintrin.h entcode.h entdec.h entenc.h fixed_generic.h float_cast.h \
